nautobot/extras/models/jobs.py

```diff
@@ -212,14 +212,25 @@ class Job(PrimaryModel):
         default=False,
         help_text="If set, the configured value will remain even if the underlying Job source code changes",
     )
+    default_job_queue_override = models.BooleanField(
+        default=False,
+        help_text="If set, the configured value will remain even if the underlying Job source code changes",
+    )
     job_queues = models.ManyToManyField(
         to="extras.JobQueue",
         related_name="jobs",
         verbose_name="Job Queues",
         help_text="The job queues that this job can be run on",
         through="extras.JobQueueAssignment",
     )
-
+    default_job_queue = models.ForeignKey(
+        to="extras.JobQueue",
+        related_name="default_for_jobs",
+        on_delete=models.PROTECT,
+        verbose_name="Default Job Queue",
+        null=False,
+        blank=False,
+    )
```

Comment:
nit: can we move `job_queues` and `default_job_queue` definition up to around line 173 so that they're kept with the other model fields rather than down here after all the `override` fields?